provisioning/security/security-provider/src/main/java/com/microsoft/azure/sdk/iot/provisioning/security/SecurityProviderX509.java

```diff
@@ -19,17 +19,16 @@
 
 public abstract class SecurityProviderX509 extends SecurityProvider
 {
-    private static final String ALIAS_CERT_ALIAS = "ALIAS_CERT";
+    private static final String CERT_ALIAS = "ALIAS_CERT";
```

Comment:
cert alias = alias cert? :) Perhaps `ALIAS_CERT`?

Reply:
```suggestion
    private static final String CERT_ALIAS = "IOTHUB_SECURITY_PROVIDER_CERT_ALIAS";
```

Reply:
I'll make this a bit more descriptive with the above suggestion